CHANGELOG.md

```diff
@@ -73,6 +103,7 @@ next available opportunity.
 * @justinpryzby for reporting an error when compressing very wide tables
 * @maxtwardowski for reporting problems with chunk exclusion and space partitions
 * @yuezhihan for reporting GROUP BY error when having compress_segmentby on an enum column
+* @maxtwardowski for reporting a in chunk exclusion for space partitions in SELECT FOR UPDATE queries
```

Comment:
this is a duplicate to the line 2 lines above
```suggestion
```

Reply:
Removed

Reply:
Removed